week-4-practise/level-2/exercise-1.js

```diff
@@ -9,17 +9,19 @@ We made a start on this for you here:
 */
 
 var bottle = {
-	volume: 0,
-	fill: function(){
-	 	// calling this function should make you bottles volume = 100; 
-	},
-	drink: function(){
-		// calling this function should decrease your bottles volume by 10;
-	},
-	empty: function(){
-		// this function should return true if your bottles volume = 0
-	}
-}
+  volume: 0,
+  fill: function() {
+    return (this.volume = 100);
+  },
+  drink: function() {
+    return (this.volume = this.volume - 10);
+  },

```

Comment:
`fill` and `drink` don't need to return anything, it's only important here that they have the right side-effect - changing the value of the bottle's `volume` property.  They do - good!